code/release-2/00-metadata/emp500_s1_merge_sample_info.ipynb

```diff
@@ -0,0 +1,457 @@
+    "df75 = pd.read_excel('/Users/luke.thompson/emp/500-metadata-studies/75-Song-aquatic-fecal/Song75_host_associated_4_samples.xlsx', skiprows=[1,2])\n",
+    "df76 = pd.read_excel('/Users/luke.thompson/emp/500-metadata-studies/76-Minich-fecal/Minich76_manure_6_samples.xlsx', skiprows=[1,2])\n",
+    "df77 = pd.read_excel('/Users/luke.thompson/emp/500-metadata-studies/77-Tait-water/Tait77_water_9_samples.xlsx', skiprows=[1,2])\n",
+    "df78 = pd.read_excel('/Users/luke.thompson/emp/500-metadata-studies/78-Tait-sediment/Tait78_sediment_3_samples.xlsx', skiprows=[1,2])\n",
+    "df81 = pd.read_excel('/Users/luke.thompson/emp/500-metadata-studies/81-Metcalf-skin/Metcalf81_skin_3_samples.xlsx', skiprows=[1,2])\n",
+    "df82 = pd.read_excel('/Users/luke.thompson/emp/500-metadata-studies/82-Uren-lichen/Uren82_plant_associated_21_samples.xlsx', skiprows=[1,2])\n",
+    "df84 = pd.read_excel('/Users/luke.thompson/emp/500-metadata-studies/84-85-86-87-Rohwer-misc/Rohwer84_soil_15_samples.xlsx', skiprows=[1,2])\n",
+    "df85 = pd.read_excel('/Users/luke.thompson/emp/500-metadata-studies/84-85-86-87-Rohwer-misc/Rohwer85_sediment_27_samples.xlsx', skiprows=[1,2])\n",
+    "df86 = pd.read_excel('/Users/luke.thompson/emp/500-metadata-studies/84-85-86-87-Rohwer-misc/Rohwer86_sediment_3_samples.xlsx', skiprows=[1,2])\n",
+    "df88 = pd.read_excel('/Users/luke.thompson/emp/500-metadata-studies/88-Mousseau-fecal/Mousseau88_host_associated_112_samples.xlsx', skiprows=[1,2])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dfs = [df0, df1, df2, df3, df5, df9, df18, df19, df21, df23, df26, df27,\n",
+    "       df33, df34, df36, df37, df38, df39, df40, df42, df43, df45, df46, \n",
+    "       df47, df50, df51, df52, df53, df54, df56, df58, df59, df62, df63, \n",
+    "       df65, df68, df71, df72, df74, df75, df76, df77, df78, df81, df82, \n",
+    "       df84, df85, df86, df88]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "49"
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "len(dfs)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Metadata columns per study"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "metadata_cols = pd.DataFrame(columns=['metadata_columns'])\n",
+    "for df in dfs:\n",
+    "    emp500_study_id = df_samp[df_samp['#SampleID'] == df.sample_name[0]]['emp500_study_id'].values[0]\n",
+    "    columns = df.columns[2:]\n",
+    "    columns = columns.drop('tier', errors='ignore').append(pd.Index(['tier']))\n",
+    "    metadata_cols.loc[emp500_study_id] = ', '.join(columns)"

```

Comment:
I don't really know what this is doing, is there an english description of what's going on?